wordpress-mobile/WordPress-Android
# Create referrers block in Stats

Fixes #8677 - the FluxC change should be merged first - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1018
`feature/init_time_stats` should be merged before this branch.
This PR is adding a referrers block to the Days/Weeks/Months/Years stats tabs. The block should contain at most 6 items, if there is more than 6, we should show the `View more` button. Some of the referrers are expandable and contain a list of sub-referrers.
I've also noticed that the `ARGS_SELECTED_DATE` makes sense for the `DWMY` stats so I started sending it to the `View all` activity.

To test:
* Open Stats
* Go to Days tab
* You see a referrers block - either with no data or with some real data
* Go to Weeks/Months/Years tabs to check that the data in the other tabs is different
* The `View more` button is only visible if there is 6 items visible
* Clicking the `View more` button shows an old full screen `View all` activity
